### PR TITLE
Vw addons

### PR DESCRIPTION
**Pull request by kupermind**
- Adding view functions for removed nominees;
- Correcting variable names, natspecs, comments.

### CODE REVIEW

File: contracts/VoteWeighting.sol [hunk 1]
```diff
@@ -159,6 +159,8 @@ contract VoteWeighting {
 
     // Set of Nominee structs
     Nominee[] public setNominees;
+    // Set of removed Nominee structs
+    Nominee[] public setRemovedNominees;
```
Comment (kupermind) [inline contracts/VoteWeighting.sol:163]: Adding a set to push removed nominees.

File: contracts/VoteWeighting.sol [hunk 2]
```diff
@@ -683,30 +685,44 @@ contract VoteWeighting {
         return setNominees.length - 1;
     }
 
+    /// @dev Get the total number of removed nominees.
+    /// @notice The zero-th default nominee Id with id == 0 does not count.
+    /// @return Total number of removed nominees.
+    function getNumRemovedNominees() external view returns (uint256) {
```
Comment (kupermind) [inline contracts/VoteWeighting.sol:691]: Number of removed nominees.

File: contracts/VoteWeighting.sol [hunk 3]
```diff
@@ -683,30 +685,44 @@ contract VoteWeighting {
...
-        nominees = setNominees;
+    /// @return Set of all the nominees in the contract.
+    function getAllNominees() external view returns (Nominee[] memory) {
+        return setNominees;
+    }
+
+    /// @dev Gets a full set of removed nominees.
+    /// @notice The returned set includes the zero-th empty nominee instance.
+    /// @return Set of all the removed nominees in the contract.
+    function getAllRemovedNominees() external view returns (Nominee[] memory) {
```
Comment (kupermind) [inline contracts/VoteWeighting.sol:705]: Simple getter for the set of removed nominees.

Comment (Tanya-atatakai) [reply contracts/VoteWeighting.sol:705]: awesome, thank you

**Review by Tanya-atatakai — APPROVED** [review]

**Review by mariapiamo — DISMISSED** [review]

**Review by mariapiamo — APPROVED** [review]

**Review by 77ph — APPROVED** [review]

**Pull request merged**